cmd/experimental/kjobctl/docs/commands/kjobctl_create/_index.md

```diff
@@ -46,6 +46,10 @@ Create a task
 	--min-replicas small-group=1 \ 
 	--max-replicas small-group=5 \ 
 	--localqueue my-local-queue-name
+
+  # Create slurm 
+  kjobctl create slurm --profile my-application-profile ./script.sh \
+	-- --array 0-5 --nodes 3 --ntasks 1
```

Comment:
```suggestion
  kjobctl create slurm --profile my-application-profile -- \
	-- --array 0-5 --nodes 3 --ntasks 1 ./script.sh
```

I think something like this would make more sense, matching how `srun` works https://rc-docs.northeastern.edu/en/latest/slurmguide/slurmrunningjobs.html

@mwielgus

Reply:
can we also have a page dedicated to how to use the slurm simulation? and what caveats to take into account. This could be a separate PR.

Reply:
I synced with @mwielgus offline and he agreed to change the order of the arguments

Reply:
> can we also have a page dedicated to how to use the slurm simulation? and what caveats to take into account. This could be a separate PR.

Created an issue #2954.

Reply:
> I think something like this would make more sense, matching how `srun` works https://rc-docs.northeastern.edu/en/latest/slurmguide/slurmrunningjobs.html
> 
> @mwielgus

Done